### ISSUE TITLE
Does not work with the latest preview

### ISSUE THREAD

**Issue by aeos** [serilog/serilog-extensions-logging#116]
Hi,

I am trying to use the Microsoft.Extensions.Hosting package 2.1.0-preview1-final. and it uses the Microsoft.Extensions.Logging package of the same version. However when I add Serilog.Extensions.Logging 2.0.2 to my project I am unable to reference:
use Serilog.Extensions.Logging;

Nor am I able to call AddSerilog() on the ILoggingBuilder

**Comment by nblumhardt** [comment]
Hi, thanks for the note! Any clues you can spot? There could be some info to be found in the output of dotnet restore, or the build process; nothing immediately comes to mind here (AFAIK there shouldn't be breaking changes that impact this library).

**Comment by aeos** [comment]
after much wrangling, I got it to work. All's well!

**Comment by andymac4182** [comment]
What did you have to do to wrangle it @aeos ?

**Comment by aeos** [comment]
I am not convinced I need to do anything. Originally vscode was not triggering intellisense for addserilog and did not allow me to use serilog.extensions.logging. so I checked out this repository, changed the dependency and compiled a dll. I included it in my project and everything worked. I then removed the dll to confirm the fix, only to find that vscode had no problems with the original nuget package. So I am thinking it was a vscode gremlin.
